sources/packages/backend/apps/api/src/route-controllers/assessment/assessment.students.controller.ts

```diff
@@ -133,10 +137,34 @@ export class AssessmentStudentsController extends BaseController {
     @Param("applicationId", ParseIntPipe) applicationId: number,
     @UserToken() userToken: StudentUserToken,
   ): Promise<RequestAssessmentSummaryAPIOutDTO[]> {
-    return this.assessmentControllerService.getPendingAndDeniedAppeals(
-      applicationId,
-      userToken.studentId,
-    );
+    const pendingAndDeniedAppeals =
+      await this.assessmentControllerService.getPendingAndDeniedAppeals(
+        applicationId,
+        userToken.studentId,
+      );
+    const inProgressAndDeclinedApplicationOfferingChangeRequests =
+      await this.applicationOfferingChangeRequestService.getApplicationOfferingChangeRequestsByStatus(
+        applicationId,
+        userToken.studentId,
+        [
+          ApplicationOfferingChangeRequestStatus.InProgressWithStudent,
+          ApplicationOfferingChangeRequestStatus.InProgressWithSABC,
+          ApplicationOfferingChangeRequestStatus.DeclinedByStudent,
+          ApplicationOfferingChangeRequestStatus.DeclinedBySABC,
+        ],
+      );
+    const studentInProgressAndDeclinedApplicationOfferingChangeRequestsArray: RequestAssessmentSummaryAPIOutDTO[] =
```

Comment:
Not a blocker, For the appeal, we are doing the dto transformation in the service (which will be specific for this endpoint), then you can do something like this 
![image](https://github.com/bcgov/SIMS/assets/77353155/2fb7b3f3-55c3-440f-aee8-d2e2b1772681)

Reply:
I agree. We can move the transformation to controller service.

Reply:
I also agree. Please notice that the comments have two points, please do both.
1. Move the transformation to the controller service.
2. Execute the calls in parallel.